### PR TITLE
ola world

### PR DESCRIPTION
**Pull request by Arthur7711**
I say Hello World

### CODE REVIEW

**Comment by KarlenNersisyan** [comment]
Good

File: wwe.js [hunk 1]
```diff
@@ -0,0 +1 @@
+console.log("Hello World");
```
Comment (KarlenNersisyan) [inline wwe.js:1]: es inchaaaaaaaaa?

**Review by KarlenNersisyan — COMMENTED** [review]
Uxxvac tarberak@ uxarki

**Comment by KarlenNersisyan** [comment]
erku lucumnerov

**Comment by Arthur7711** [comment]
okay, thank you , bey

**Pull request merged**